Create pod eviction concept topic (#27739)
kubernetes/website: content/en/docs/reference/glossary/graceful-eviction.md

---

> title: Graceful eviction
> id: graceful-eviction

Reviewer: I want to use the word “exogenous”, but it's too complicated.
```suggestion
title: API-initiated eviction
```
How about this?

date: 2021-04-27

> full_link: /docs/concepts/scheduling-eviction/pod-eviction/

Reviewer: Maybe add a fragment identifier to the end of this URL?a

short_description: >

>   Graceful eviction is the process by which you create an `Eviction` object using

Reviewer: I'm not sure that backticks (and other Markdown formatting) in a short description will work how you might expect. I'd remove them unless we're confident they always render OK.

>   Graceful eviction is the process by which use the Eviction API to create an

Author: :( I blame Friday. Thanks for the catch!

  the Eviction API to gracefully terminate pods.
aka:
tags:
- operation

> - fundamental

Reviewer: I'd omit this one from `fundamental`.

---

> Graceful eviction is the process by which you use the [Eviction API](/docs/reference/generated/kubernetes-api/v1.21/#create-eviction-pod-v1-core)

Reviewer: ```suggestion
Graceful eviction is the process by which you use the [Eviction API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#create-eviction-pod-v1-core)
```
this then tracks the version we're documenting.

<!--more-->

> You can request graceful eviction either by directly calling the Eviction API 
> using a client of the kube-apiserver, like the `kubectl drain` command. 
> When an `Eviction` object is created, the API server terminates the Pod. 

Reviewer: ```suggestion
You can request graceful eviction either by directly calling the Eviction API 
using a client of the kube-apiserver, like the `kubectl drain` command. 
When an `Eviction` object is created, the API server terminates the Pod.

API-initiated eviction is not the same as
[node-initiated eviction](/docs/concepts/scheduling-eviction/pod-eviction/#kubelet-eviction).
```
